src/App.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div>
+    <h1>Todo List</h1>
+    <AddTodo />
+    <hr>
+    <todoList />
+  </div>
+</template>
+<script>
+import TodoList from './components/TodoList.vue'
```

Comment:
Can we use absolute path?

Reply:
This happens to people who don't use semicolons in javascript.
```typescript
(this as any).start = function() {
  console.log('start')
}

(this as any).finish = function() {
  console.log('finish');
}

finish();
```

what's the output?

Reply:
it should be 'finish' because ASI set semicolon after {} and  console.log('start')

Reply:
```
start
TypeError: Cannot set property 'finish' of undefined
```